src/client/analytics/PlaytestData.tsx

```diff
@@ -89,7 +93,7 @@ const PlaytestData: React.FC<PlaytestDataProps> = ({ cubeAnalytics }) => {
             { key: 'mainboards', title: 'Mainboard Count', sortable: true, heading: false },
           ]}
           data={data}
-          sortFns={{ label: compareStrings }}
+          sortFns={{ card: compareCardNames }}
```

Comment:
The card name column has key `card` so the sort function for label did nothing. Even if it was used, that would have compared the object as a string which would have been `[object Object]` and thus no sorting either.